arch/xtensa/core/gdbstub.c

```diff
@@ -705,7 +705,9 @@ size_t arch_gdb_reg_readone(struct gdb_ctx *ctx, uint8_t *buf, size_t buflen,
 	int idx;
 	size_t ret;
 
-	ret = 0;
+	buf[0] = 'x';
+	buf[1] = 'x';
+	ret = 2;
```

Comment:
Unclear what's happening here?

Reply:
This is to make unknown registers show as unavailable instead of returning an error. xt-gdb requests a handful of registers that aren't on the list which would otherwise cause the stub to return an error, breaking things like `info reg`. While those registers could be added, I no longer have access to XCC so I can't do it myself.